<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdg_tasks</class>
 <widget class="QWidget" name="wdg_tasks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="wdg_tasks_hor_layout_1">
     <item>
      <widget class="QLabel" name="lbl_sort_by">
       <property name="text">
        <string>Sort by:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbo_sort_by"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lbl_task_count">
       <property name="text">
        <string>0 pending tasks</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="wdg_tasks_hor_layout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_start_task">
       <property name="text">
        <string>Start Task</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_close_task">
       <property name="text">
        <string>Close Task</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QListWidget" name="lvw_tasks"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
